lib/inspec/plugin/v2/gem_source_manager.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require "singleton" unless defined?(Singleton)
+require "forwardable" unless defined?(Forwardable)
```

Comment:
Do we need to use `forwardable` here? Does not seem like we are using it currently here.

Reply:
yes thanks for highlighting.. modified to use delegate